### PR TITLE
! 9.0 TASK: Unify variable and parameter names (needs DB schema update)

### PR DESCRIPTION
**Pull request by bwaidelich**
!! when updating a production instance, run ./flow doctrine:migrate - this will update your existing projections (and you do not need to replay all for this).

Renames variable and parameter names:
* `contentRepositoryIdentifier` => `contentRepositoryId`
* `identifier` => `nodeAggregateId`
* contentStreamIdentifier -> contentStreamId
* adds DB migration for projection change

### CODE REVIEW

**Comment by ahaeslich** [comment]
Out of curiosity: Are you planning to adjust the inconsistent use of `contentStreamId`/`contentStreamIdentifier` and `nodeAggregateId`/`nodeAggregateIdentifier` as well?

**Comment by bwaidelich** [comment]
> Are you planning to adjust the inconsistent use of contentStreamId/contentStreamIdentifier and nodeAggregateId/nodeAggregateIdentifier as well?

Good catch, maybe.. But I hope that someone is faster than me :D

File: Neos.ContentRepositoryRegistry/Classes/Factory/EventStore/DoctrineEventStoreFactory.php [hunk 1]
```diff
@@ -24,8 +24,8 @@ public function build(ContentRepositoryId $contentRepositoryIdentifier, array $c
         );
```
Comment (ahaeslich) [inline Neos.ContentRepositoryRegistry/Classes/Factory/EventStore/DoctrineEventStoreFactory.php:24]: Was it intentional that you didn't update the argument `$contentRepositoryIdentifier` of the `build` function here and in the interface?

Comment (skurfuerst) [reply Neos.ContentRepositoryRegistry/Classes/Factory/EventStore/DoctrineEventStoreFactory.php:24]: thanks, fixed :)

**Comment by skurfuerst** [comment]
FYI working on remaining replacements :)

**Comment by skurfuerst** [comment]
Needed migration: https://github.com/neos/neos-development-collection/pull/4127/files#diff-26088689b2a296b2ecf6c41d175a39b14940ff5f3d25931585f954d71da2cd95

**Comment by skurfuerst** [comment]
finished from my side @bwaidelich @ahaeslich . Contains a doctrine migration for updating production databases without downtime / re-applying events. Will merge once CI is green. https://github.com/neos/neos-ui/pull/3435 needs to be merged alongside.

**Pull request merged**